packages/editor/src/components/document-tools/style.scss

```diff
@@ -74,14 +74,8 @@
 }
 
 .editor-document-tools .editor-document-tools__left > .editor-document-tools__inserter-toggle.has-icon {
-	min-width: $button-size-compact;
-	width: $button-size-compact;
-	height: $button-size-compact;
-	padding: 0;
```

Comment:
The padding doesn't seem to be necessary?

Reply:
Not needed, yes, the button is unchanged as the styles currently come from here:

https://github.com/WordPress/gutenberg/blob/dd77795ba486f60cab31ede0d1e4c7b598303f43/packages/editor/src/components/document-tools/style.scss#L39-L48